### PR TITLE
Use ansible-runner change to get periodic keep-alive messages in K8S

### PR DESCRIPTION
**Pull request by AlanCoding**
##### SUMMARY
This is a candidate patch for consuming https://github.com/ansible/ansible-runner/pull/1191, which adds keepalive messages.

FYI for @nitzmahone, I hope we are narrowing in on this implementation. This will require keeping and supporting the environment variable. There are no plans to use the CLI flag. I also maintain a weak preference to have the event callback pass on the keep-alive events.

##### ISSUE TYPE
 - New or Enhanced Feature

##### COMPONENT NAME
 - API

### CODE REVIEW

**Comment by AlanCoding** [comment]
I tested building the UI, and then I was able to edit and save the value of the setting. Without UI changes I was able to see the setting, but not able to edit it. I know some changes here are just test file changes.

**Comment by mabashian** [comment]
UI changes lgtm

File: awx/main/conf.py [hunk 1]
```diff
@@ -282,6 +282,16 @@
     placeholder={'HTTP_PROXY': 'myproxy.local:8080'},
 )
 
+register(
+    'AWX_RUNNER_KEEPALIVE_SECONDS',
+    field_class=fields.IntegerField,
+    label=_('K8S Ansible Runner Keep-Alive Message Interval'),
+    help_text=_('Only applies to K8S deployments and container_group jobs. If not 0, send a message every so-many seconds to keep connection open.'),
```
Comment (shanemcd) [inline awx/main/conf.py]:
```suggestion
    help_text=_('Only applies to jobs running in a Container Group. If not 0, send a message every so-many seconds to keep connection open.'),
```

File: awx/main/tasks/callback.py [hunk 2]
```diff
@@ -85,6 +85,8 @@ def event_handler(self, event_data):
         # which generate job events from two 'streams':
         # ansible-inventory and the awx.main.commands.inventory_import
         # logger
+        if event_data.get('event') == 'keepalive':
+            return
```
Comment (AlanCoding) [inline awx/main/tasks/callback.py:89]: From discussion with @TheRealHaoLiu, we are unlikely to need this, but I would prefer to keep it in anyway. Maybe leave a comment that it should be dead code.

File: awx/main/conf.py [hunk 3]
```diff
@@ -282,6 +282,16 @@
     placeholder={'HTTP_PROXY': 'myproxy.local:8080'},
 )
 
+register(
+    'AWX_RUNNER_KEEPALIVE_SECONDS',
```
Comment (AlanCoding) [inline awx/main/conf.py:286]: should we rename this to something about K8S_?

**Review by CFSNM — APPROVED** [review]

**Pull request merged**